.travis.yml

```diff
@@ -335,6 +335,7 @@ matrix:
             - ant $OPTS build
           script:
             - ant $OPTS -f extide/o.apache.tools.ant.module test
+            - ant $OPTS -f extide/gradle test
```

Comment:
👍

Reply:
Well I finally I had to remove this one, as it seems we have a circular cluster dependency on tests, so it fails. I need to fix that in a different PR. But definitely thanks for pointing this out!